src/Components/WebView/WebView/src/WebViewManager.cs

```diff
@@ -199,7 +205,8 @@ private static Uri EnsureTrailingSlash(Uri uri)
         record RootComponent
```

Comment:
Unrelated, but there's not a very good reason for this to be a record.

Reply:
It's private - is there any drawback?

Reply:
It generates more IL (the comparer and copy ctor) that we aren't using. Just having this be a regular class would have no impact on the caller.

Reply:
OK, changed to `class`.